app/sidecar/base.toml

```diff
@@ -78,7 +78,7 @@ task-slots = ["sys"]
 
 [tasks.net]
 name = "task-net"
-stacksize = 6040
+stacksize = 8000
```

Comment:
Can you bump the stack size in `app/demo-stm32h7-nucleo/app-h753.toml` and `app/demo-stm32h7-nucleo/app-h743.toml` as well?